src/components/molecules/Supplier/AddSupplierData/index.tsx

```diff
@@ -11,10 +11,10 @@ import {
   WrapItem,
 } from '@chakra-ui/react';
 import { useForm } from 'react-hook-form';
-import AddSupplierButtons from 'components/Supplier/AddSupplierButtons';
-import breakpoints from 'styled/Supplier/AppSupplierData/breakpoints';
-import breakpointsFontSize from 'styled/Supplier/AppSupplierData/breakpointsFontSize';
-import breakpointsMargin from 'styled/Supplier/AppSupplierData/breakpointsMargin';
+import AddSupplierButtons from '../AddSupplierButtons/index';
+import breakpoints from '../styled/AddSupplierData/breakpoints';
+import breakpointsFontSize from '../styled/AddSupplierData/breakpointsFontSize';
+import breakpointsMargin from '../styled/AddSupplierData/breakpointsMargin';
 
 const AddSupplierData = () => {
```

Comment:
In the future this component should be rewritten for inputs that @dominikmlynarczyk prepared.

Reply:
It will be corrected